### PR TITLE
[Cases] Deprecated endpoints

### PR DESCRIPTION
**Pull request by cnasikas**
This PR deprecates the following cases endpoints:

- [GET: /api/cases/{case_id}/comments](https://www.elastic.co/guide/en/security/current/cases-api-get-all-case-comments.html)
- [GET: /api/cases/<case ID>/user_actions](https://www.elastic.co/guide/en/security/current/cases-api-get-case-activity.html)
- [GET: /api/cases/status](https://www.elastic.co/guide/en/security/current/cases-api-get-status.html)

It also deprecates the `includeComments` query parameter from the endpoint [GET /api/cases/{case_id}](https://www.elastic.co/guide/en/security/current/cases-api-get-case.html)

### CODE REVIEW

File: docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc [hunk 1]
```diff
@@ -5,6 +5,8 @@ Returns all user activity for the specified case.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
+WARNING: This endpoint is deprecated and it will not be supported in future releases.
```
Comment (jonathan-buttner) [inline docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: What do you think about:

```suggestion
WARNING: This endpoint is deprecated and will be removed in future releases.
```

File: docs/cases/api/cases-api/cases-api-get-case.asciidoc [hunk 2]
```diff
@@ -24,6 +24,8 @@ The URL must include the `case ID` of the case you are retrieving. Call
 returned. |No, defaults to `true`.
 |==============================================
 
+WARNING: The `includeComments` query parameter is deprecated and it will not be supported in future releases.
```
Comment (jonathan-buttner) [inline docs/cases/api/cases-api/cases-api-get-case.asciidoc:27]: I'm torn on whether to use `not supported` or `removed` for this one 🤔

**Comment by jmikell821** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/cases/api/cases-api/cases-api-get-case.asciidoc (hunk 2 above)
Comment (jmikell821) [inline docs/cases/api/cases-api/cases-api-get-case.asciidoc:27]:
```suggestion
WARNING: The `includeComments` query parameter is deprecated and will not be supported in future releases. 
```

File: docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc [hunk 3]
```diff
@@ -5,6 +5,8 @@ Returns all user activity for the specified case.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
+WARNING: This endpoint is deprecated and will be removed in future releases.
```
Comment (nastasha-solomon) [inline docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: If we can't offer a concrete statement about when the endpoint will be removed, can we state the last version it's supported on?

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for updating this @cnasikas ! I just left one question, but it applies to all of the updates you've made.

File: docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc (hunk 3 above)
Comment (cnasikas) [reply docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: @nastasha-solomon Ok! Any suggestions? Thanks!

**Review by jmikell821 — COMMENTED** [review]
One small nit - otherwise looks good!

File: docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: If there are no legal constraints or other limitations to consider, maybe something like, "this endpoint is not available after version 8.1" or "this endpoint is available until version 8.1".

Comment (cnasikas) [reply docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: > If there are no legal constraints or other limitations to consider
I am not aware of any. Do you have something in mind?

Comment (nastasha-solomon) [reply docs/cases/api/cases-api/cases-api-get-case-activity.asciidoc:8]: Gotcha. Just recommending a minor edit then! Thanks again for your work on this!

File: docs/cases/api/cases-api/cases-api-get-case-comments.asciidoc [hunk 4]
```diff
@@ -5,6 +5,8 @@ Returns all comments for the specified case.
 
 NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
 
+WARNING: This endpoint is deprecated and will be removed in future releases.
```
Comment (nastasha-solomon) [inline docs/cases/api/cases-api/cases-api-get-case-comments.asciidoc:8]:
```suggestion
WARNING: This endpoint is deprecated and will be removed in a future release.
```

Comment (nastasha-solomon) [reply docs/cases/api/cases-api/cases-api-get-case-comments.asciidoc:8]: Changing it to a singular release instead of multiple ones.

**Pull request merged**